### DISCUSSION TITLE
Revit 2023: Properties not showing when installing RevitLookup

### DISCUSSION THREAD

**Discussion by rvs1977** [jeremytammik/RevitLookup#247] [Q&A] [answered]
![2024-05-29 11_31_53-Window](https://github.com/jeremytammik/RevitLookup/assets/3578157/d0896543-becc-4d3c-ac65-d8a55e6f6750)

**Comment by Nice3point** [comment, marked as answer]
Just disable hardware acceleration
![изображение](https://github.com/jeremytammik/RevitLookup/assets/20504884/1a619459-dd38-444f-b2bc-d4255a02a7cb)

Autodesk has fixed this issue in newer versions of Revit, for 2023 disabling this setting should help

**Comment by rvs1977** [comment]
Ah, thanky! it works
